src/views/Projects/index.jsx

```diff
@@ -2,26 +2,42 @@ import { hot } from 'react-hot-loader';
 import { Component, Fragment } from 'react';
 import { withStyles } from '@material-ui/core/styles';
 import Grid from '@material-ui/core/Grid';
-import TextField from '@material-ui/core/TextField';
 import Typography from '@material-ui/core/Typography';
+import AppBar from '@material-ui/core/AppBar';
 import projects from '../../data/loader';
 import ProjectCard from '../../components/ProjectCard';
+import SearchBox from '../../components/SearchBox';
 
 @hot(module)
 @withStyles(theme => ({
   container: {
-    paddingRight: 15,
-    paddingLeft: 15,
-    marginRight: 'auto',
-    marginLeft: 'auto',
+    paddingTop: 1 * theme.spacing.unit,
+    backgroundColor: theme.palette.primary.light,
+    minHeight: `calc(100vh - 180px - ${3 * theme.spacing.unit}px)`,
+    marginTop: `calc(180px + ${theme.spacing.unit}px)`,
   },
   header: {
     paddingTop: theme.spacing.unit,
-    paddingBottom: 2 * theme.spacing.unit,
+    height: 180,
+    background: 'linear-gradient(45deg, #364598, #6AE5F4 90%)',
   },
   search: {
     marginBottom: 4 * theme.spacing.unit,
   },
+  highlightedText: {
+    backgroundColor: theme.palette.common.black,
+    color: theme.palette.primary.main,
+  },
+  flexContainer: {
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  '@media (min-width: 1024px)': {
+    container: {
+      paddingRight: 100,
+      paddingLeft: 100,
+    },
```

Comment:
We don't need `@media (min-width: 1024px)` if we just use the padding specified in the other comment to the `container` class.

Reply:
@helfi92 i think we still need it to add padding when the device is big, so the card does not look too wide, wdyt?

Reply:
👍 ok!